apps/preview/next/pages/showcases/MegaMenu/MegaMenuNavigation.tsx

```diff
@@ -0,0 +1,641 @@
+              onClick={handleOpenMenu([])}
+              variant="tertiary"
+              square
+              aria-label="Close menu"
+              className="block md:hidden mr-5 bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+            >
+              <SfIconMenu className="text-white" />
+            </SfButton>
+            <a
+              href="/"
+              aria-label="SF Homepage"
+              className="flex items-center mr-2 text-white md:mr-10 focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+            >
+              <picture>
+                <source srcSet={brandLogo.src} media="(min-width: 1024px)" />
+                <img src={brandLogoSign.src} alt="Sf Logo" className="w-8 h-8 lg:w-[12.5rem] lg:h-[1.75rem]" />
+              </picture>
+            </a>
+          </div>
+          <div className="flex flex-nowrap">
+            {actionItems.map((actionItem) => (
+              <SfButton
+                className="ml-2 text-white bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+                key={actionItem.ariaLabel}
+                aria-label={actionItem.ariaLabel}
+                variant="tertiary"
+                slotPrefix={actionItem.icon}
+                square
+              >
+                {actionItem.role === 'login' && <p className="hidden md:inline-flex">{actionItem.label}</p>}
+              </SfButton>
+            ))}
+          </div>
+        </div>
+        {/* Desktop dropdown */}
+        <nav ref={refs.setFloating}>
+          <ul
+            className="hidden md:flex px-6 py-2 bg-white border-b border-b-neutral-200 border-b-solid"
+            onBlur={handleBlurWithin}
+          >
+            {content.children.map((node) => (
+              <li key={node.key}>
+                <SfButton
+                  variant="tertiary"
+                  onMouseEnter={handleOpenMenu([node.key])}
+                  onClick={handleOpenMenu([node.key])}
+                  className="mr-2"
+                >
+                  <span>{node.value.label}</span>
+                  <SfIconChevronRight className="rotate-90" />
+                </SfButton>
+
+                {isOpen && activeNode.length === 1 && activeNode[0] === node.key && (
+                  <div
+                    key={activeMenu.key}
+                    style={style}
+                    className="hidden md:grid gap-x-6 grid-cols-4 bg-white shadow-lg p-6 left-0 right-0"
+                    onMouseLeave={close}
+                  >
+                    {activeMenu.children.map((node) =>
+                      node.isLeaf ? (
+                        <Fragment key={node.key}>
+                          <div className="mb-2">
+                            <SfListItem
+                              as="a"
+                              size="sm"
+                              role="none"
```

Comment:
Why role="none"? This element still acts as an anchor, not only for presentation, right?

Reply:
It's leftover from the implementation I used as a staring point. I'll remove it.